jenkins_pipelines/manager_prs/Jenkinsfile_prs_ruby_rubocop

```diff
@@ -51,12 +56,17 @@ pipeline {
             }
         }
         stage('Run tests against PR') {
+            when {
+                anyOf {
+                    changeset "**/*.rb"
```

Comment:
That will not take into account Rakefile

Reply:
In GH actions we have it like that: https://github.com/SUSE/spacewalk/blob/6fbfe9b94e87a0c1b0d025c3a266fc84c396b3ba/.github/workflows/rubocop.yml#L5

I followed this convention, in the recent commit also for `yml` files. I have no problem to add `Gemfile` and `Rakefile` if needed.

Reply:
Good catch! I think we should include both Gemfile and Rakefile in both GH Actions and Jenkinsfile. As few days ago, we had an issue in Rakefile if I recall correctly. Would you mind to take care of both changes please?

Reply:
Done